onnxmltools/utils/metadata_props.py

```diff
@@ -0,0 +1,68 @@
+from distutils.version import StrictVersion
+
```

Comment:
Please remove the empty line here and move `from distutils.version import StrictVersion ` to line 4.